README.md

```diff
@@ -4,22 +4,116 @@
+
+## Quickstart
+
+After [installing semgrep](#installation), run the default set of checks with:
+
+```bash
+semgrep --config=r2c path/to/repo
+```
 
-**Try it now:** [https://semgrep.live](https://semgrep.live/)
+Or try Semgrep in your browser: [https://semgrep.live](https://semgrep.live/).
 
 ## Overview
 
-Language support:
+### Key features
+
+* **Language aware**: Semgrep parses source code into Abstract Syntax Trees (ASTs), so it understands function calls, method and class definitions, conditionals, and more. 
+  * Unlike regexes, Semgrep won't get tripped up matching things you don't care about in comments or string literals.
+* **Fast**: Speedy enough to run on every build, commit, or file save.
+* **Batteries included**: Leverage hundreds of preexisting security checks for popular languages and frameworks in the community [registry](https://semgrep.live/r), including coverage for the [OWASP Top 10](https://owasp.org/www-project-top-ten/).
+* **Handles multiple languages**: No need to learn and maintain multiple tools for your polyglot environment (e.g. ESLint, find-sec-bugs, rubocop, gosec, ...).
+* **Easy to tweak**: Rules look like the code you’re searching, no static analysis PhD required.
+* Does **not** require compilation, Semgrep runs on source code directly. 
+* **Easy to integrate**: Highly portable and many CI and git hook integrations already exist. Run via CLI or Docker, output `--json` and pipe results into whatever system you want.
+
+
+### Language support
+
+| **Python** | **Javascript** | **Go &nbsp; &nbsp; &nbsp;** | **Java &nbsp;** | **C &nbsp; &nbsp; &nbsp; &nbsp;** | **Ruby** | **Typescript** | **PHP &nbsp; &nbsp;** |
+|:-----------|:---------------|:----------------------------|:----------------|:----------------------------------|:---------|:---------------|:----------------------|
+| ✅          | ✅              | ✅                           | ✅               | ✅                                 | 🚧 |  Coming...      | Coming...             |
+
+... and many more coming.
+
+### Pattern Syntax Teaser
+
+One of the most unique and useful things about Semgrep is how easy it is to write and iterate on queries. 
 
-| **Python** | **Javascript** | **Go &nbsp; &nbsp; &nbsp;** | **Java &nbsp;** | **C &nbsp; &nbsp; &nbsp; &nbsp;** | **Typescript** | **PHP &nbsp; &nbsp;** |
-|:-----------|:---------------|:----------------------------|:----------------|:----------------------------------|:---------------|:----------------------|
-| ✅          | ✅              | ✅                           | ✅               | ✅                                 | Coming...      | Coming...             |
+The goal is to make it as *easy as possible* to go from an idea in your head to finding the code patterns you intend to.
 
-Example patterns:
+**Example**: Let's say you want to find all calls to a function named `exec`, and you don't care about the arguments. With Semgrep, you could simply supply the pattern `exec(...)` and you'd match:
```

Comment:
```suggestion
**Example**: Say you want to find all calls to a function named `exec`, and you don't care about the arguments. With Semgrep, you could simply supply the pattern `exec(...)` and you'd match:
```

I avoid using the first person plural in documentation / instructions as it can get fairly messy.